packages/amazonq/src/lsp/auth.ts

```diff
@@ -114,11 +115,14 @@ export class AmazonQLspAuth {
             .setProtectedHeader({ alg: 'dir', enc: 'A256GCM' })
             .encrypt(encryptionKey)
 
+        const conn = AuthUtil.instance.conn
+        const startUrl = conn?.type === 'sso' ? conn.startUrl : undefined
```

Comment:
can we just use AuthUtil.instance.startUrl here? I think previously we were using `AuthUtil.instance.auth.startUrl` which might have been causing the problem?

Reply:
Yeah that was the root, of it. I haven't updated the description but AuthUtil.instance.auth.startUrl is actually using `Auth` and not the one from `AuthUtil`. It looks like we never set an `activeConnection` in `Auth` so `startUrl` from there returns nothing.